include/zephyr/net/ieee802154_radio.h

```diff
@@ -779,12 +1510,27 @@ struct ieee802154_radio_api {
 	 * @note requires IEEE802154_HW_TXTIME and/or IEEE802154_HW_RXTIME
 	 * capabilities.
 	 *
+	 * @note This is an alias for `radio_api->get_time_reference()
+	 * ->get_time()` for drivers that support the network uptime reference
+	 * API.
+	 *
 	 * @param dev pointer to IEEE 802.15.4 driver device
 	 *
 	 * @return nanoseconds relative to the network subsystem's local clock
 	 */
 	net_time_t (*get_time)(const struct device *dev);
 
+	/**
+	 * @brief Provides access to the full API of the underlying network
+	 * uptime reference of the driver.
+	 *
+	 * @param dev pointer to IEEE 802.15.4 driver device
+	 *
+	 * @returns a pointer to the driver's underlying network uptime
+	 * reference or NULL if not supported.
+	 */
+	const struct net_time_reference_api *(*get_time_reference)(const struct device *dev);
```

Comment:
Any reference for `struct net_time_reference_api`? Is it proposed in some other PR?

Reply:
OMG, this should not have slipped in here. That comes from a downstream commit. Thx for catching this!